bodhi/server/models/models.py

```diff
@@ -854,6 +854,7 @@ def edit(cls, request, data):
             comment += '\n\nRemoved build(s):\n'
             for removed_build in removed_builds:
                 comment += "\n- %s" % removed_build
+            comment += '\n\n\nKarma has been reset.'
```

Comment:
That can be done..But anyhow the text is going to appear at the end(After couple of lines) which means it applies both for `new_builds` and `removed_builds`. But for consistency and clarification we can have it in both the places.